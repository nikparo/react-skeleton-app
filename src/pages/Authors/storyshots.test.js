import initStoryshots from '@storybook/addon-storyshots';

// Getting automatic snapshots for Storybook stories seems like a win-win.
// Limiting storyshots to work on a component by component basis, for better
// insight into what broke and who (CODEOWNERS) may need to know.

initStoryshots({
  storyKindRegex: /^Authors($|\/)/,
});
