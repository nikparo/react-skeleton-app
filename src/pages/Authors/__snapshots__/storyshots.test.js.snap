// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Authors Main 1`] = `
<div
  className="sc-gsDKAQ iJqAtc"
>
  <nav
    className="sc-jrQzAO ehfRWL"
  >
    <div
      className="sc-gsDKAQ iJqAtc"
    >
      <input
        className="sc-hKwDye ivQcNy"
        onChange={[Function]}
        placeholder="Fuzzy author filter..."
        style={
          Object {
            "flex": "1 1 auto",
          }
        }
        type="text"
        value=""
      />
      <button
        className="sc-dkPtRN kKnTFd"
        disabled={true}
        onClick={[Function]}
        type="button"
      >
        X
      </button>
    </div>
    <ul
      className="sc-eCImPb jrNcHt"
    >
      <li
        className="sc-jRQBWg jLxfDb"
      >
        No authors found
      </li>
    </ul>
  </nav>
  <div
    className="sc-kDTinF jRSqQa"
  >
    <div
      className="sc-gsDKAQ iJqAtc"
    >
      <button
        className="sc-dkPtRN kKnTFd"
        onClick={[Function]}
        title="Order descending"
      >
        ↑
      </button>
      <button
        className="sc-dkPtRN kKnTFd"
        onClick={[Function]}
        title="Order ascending"
      >
        ↓
      </button>
      <div
        style={
          Object {
            "flex": "1 1 auto",
          }
        }
      />
      <input
        className="sc-hKwDye ivQcNy"
        onChange={[Function]}
        placeholder="Fuzzy post filter..."
        type="text"
        value=""
      />
      <button
        className="sc-dkPtRN kKnTFd"
        disabled={true}
        onClick={[Function]}
        type="button"
      >
        X
      </button>
    </div>
    <p
      className="sc-bdvvtL iIgXNb"
    >
      Select an author to view their posts
    </p>
  </div>
</div>
`;

exports[`Storyshots Authors/UI UI 1`] = `
<div
  className="sc-gsDKAQ iJqAtc"
>
  <nav
    className="sc-jrQzAO ehfRWL"
  >
    <div
      className="sc-gsDKAQ iJqAtc"
    >
      <input
        className="sc-hKwDye ivQcNy"
        onChange={[Function]}
        placeholder="Fuzzy author filter..."
        style={
          Object {
            "flex": "1 1 auto",
          }
        }
        type="text"
        value="Sam"
      />
      <button
        className="sc-dkPtRN kKnTFd"
        disabled={false}
        onClick={[Function]}
        type="button"
      >
        X
      </button>
    </div>
    <ul
      className="sc-eCImPb jrNcHt"
    >
      <li
        className="sc-jRQBWg jLxfDb"
      >
        No authors found
      </li>
    </ul>
  </nav>
  <div
    className="sc-kDTinF jRSqQa"
  >
    <div
      className="sc-gsDKAQ iJqAtc"
    >
      <button
        className="sc-dkPtRN kKnTFd"
        onClick={[Function]}
        title="Order descending"
      >
        ↑
      </button>
      <button
        className="sc-dkPtRN kKnTFd"
        onClick={[Function]}
        title="Order ascending"
      >
        ↓
      </button>
      <div
        style={
          Object {
            "flex": "1 1 auto",
          }
        }
      />
      <input
        className="sc-hKwDye ivQcNy"
        onChange={[Function]}
        placeholder="Fuzzy post filter..."
        type="text"
        value=""
      />
      <button
        className="sc-dkPtRN kKnTFd"
        disabled={true}
        onClick={[Function]}
        type="button"
      >
        X
      </button>
    </div>
    <p
      className="sc-gKclnd fpuXWz"
    >
      Error fetching posts
    </p>
    <p
      className="sc-bdvvtL iIgXNb"
    >
      Select an author to view their posts
    </p>
  </div>
</div>
`;
